pandas/core/arrays/floating.py

```diff
@@ -127,13 +127,19 @@ class FloatingArray(NumericArray):
 
 This dtype uses ``pd.NA`` as missing value indicator.
 
-Attributes
-----------
-None
```

Comment:
this one (and methods) might need to stay

Reply:
Done, thanks